src/power.rs

```diff
@@ -56,6 +138,27 @@ impl Power {
     }
 }
 
+macro_rules! wakeup_pins {
+    ($($PIN:path: $line:expr,)+) => {
+        $(
+            impl<M> Into<WakeUpLine> for &$PIN {
```

Comment:
Hmm clippy thinks that it is better to use `From`.

Reply:
Yup, fixed!